app/src/main/java/com/it3048/mobiledeviceproject/RetrofitInstance.kt

```diff
@@ -5,7 +5,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "PLACEHOLDER"
+    private const val BASE_URL = "PLACEHOLDER"
```

Comment:
changed variable to constant to follow coding conventions.
https://developer.android.com/kotlin/style-guide#naming_2